src/api/studyStats.ts

```
@@ -25,7 +25,10 @@ function parseStatsTable(table: HTMLTableElement): Record<string, string> {
     for (const row of Array.from(table.querySelectorAll('tr'))) {
         const cells = row.querySelectorAll('td');
         if (cells.length >= 2) {
-            const key = (cells[0].textContent || '').trim();
+            // IS labels use a non-breaking space ( ) after single-letter
```

ESLint

[error] Irregular whitespace not allowed.

(no-irregular-whitespace)